server/__init__.py

```diff
@@ -379,6 +380,8 @@ def create_app(nl_root=DEFAULT_NL_ROOT):
   if os.path.isfile(BLOCKLIST_SVG_FILE):
     with open(BLOCKLIST_SVG_FILE) as f:
       blocklist_svg = json.load(f) or []
+  else:
+    blocklist_svg = ["dc/g/Uncategorized", "oecd/g/OECD"]
```

Comment:
Just want to make sure, do we want Uncategorized and OECD blocked by default?

Reply:
Since we don't show them in prod, we should not show them in custom DC, which is this line covers.